test/HealthChecks.Npgsql.Tests/Functional/TestLogger.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.Extensions.Logging;
+
+namespace HealthChecks.Npgsql.Tests.Functional;
+
+public sealed class TestLogger : ILogger
```

Comment:
I suggest to move `TestLogger` and `TestLoggerProvider` into `test` folder and and add `Compile Include="..."` in `Directory.Build.props` for each test project.

Reply:
See `<ItemGroup Condition="$(MSBuildProjectName.EndsWith('.Tests')) OR $(MSBuildProjectName) == 'UnitTests'">` in `Directory.Build.props`.

Reply:
done